aspnetcore/grpc/browser.md

```diff
@@ -20,37 +20,21 @@ It is not possible to call a HTTP/2 gRPC service from a browser-based app. [gRPC
 
 ## Configure gRPC-Web in ASP.NET Core
 
-gRPC services hosted in ASP.NET Core can be configured to support gRPC-Web alongside HTTP/2 gRPC. gRPC-Web does not require any changes to services, the only modification is startup configuration.
+gRPC services hosted in ASP.NET Core can be configured to support gRPC-Web alongside HTTP/2 gRPC. gRPC-Web does not require any changes to services. The only modification is startup configuration.
 
 To enable gRPC-Web with an ASP.NET Core gRPC service:
 
 1. Add a reference to the [Grpc.AspNetCore.Web](https://www.nuget.org/packages/Grpc.AspNetCore.Web) package.
 2. Configure the app to use gRPC-Web by adding `AddGrpcWeb(...)` and `UseGrpcWeb()` to *Startup.cs*:
```

Comment:
```suggestion
* Configure the app to use gRPC-Web by adding `AddGrpcWeb` and `UseGrpcWeb` to *Startup.cs*:
```

Reply:
That's only a suggestion.  Check the code to see what's actually called. I don't think `UseGrpcWeb` is called.

You need to make two sample's, the one he has and an alternate sample.

Reply:
Verified: UserGrpWeb() is called in the current full browser app sample.

Reply:
But it's not called in that snippet. You're doing two snippets - right?

Reply:
In the original PR UseGrpWeb() is called:
 "// Add gRPC-Web middleware after routing and before endpoints
    app.UseGrpcWeb();"

In the full app sample it is also called in startup.cs
"app.UseGrpcWeb();"

Sidenote: I will be replacing the the small non-compilable snippet with the full startup.cs example in a moment.  Not ready for review yet.